pl.art.lach.slawek.apps.V2BlankBrowser.appdata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Sławomir Lach <slawek@lach.art.pl> -->
+<component type="desktop-application">
+  ​  <id>pl.art.lach.slawek.apps.V2BlankBrowser</id>
```

Comment:
this doesn't seem to be a valid id.

`apps.slawek.lach.art.pl` doesn't exist and the website linked is `rcgstore.pl`

Reply:
I own subdomain slawek.lach.art.pl .

Reply:
so `pl.art.lach.slawek.V2BlankBrowser`